app/views/layouts/application.html.haml

```diff
@@ -1,11 +1,15 @@
 !!!
 %html
   %head
-    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
+    %meta{ content: "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```

Comment:
> __[HAML-Lint]__ Style/StringHashKeys: Prefer symbols instead of strings as hash keys. (https://rubystyle.guide#symbols-as-keys) ([link](https://sider.review/gh/repos/209740775/pulls/31#issue-4606870))

fixed it